packages/main/src/themes/TableRow.css

```diff
@@ -90,4 +90,18 @@
 :host([selected][type="Active"]) .ui5-table-row-root:hover:not(:active),
 :host([selected][mode="SingleSelect"]) .ui5-table-row-root:hover:not(:active) {
 	background-color: var(--sapList_Hover_SelectionBackground);
-}
\ No newline at end of file
+}
+
+/* Navigated */
+:host([navigated]) tr,
+:host([navigated]) tr[data-ui5-dir="ltr"] {
+	box-sizing: border-box;
+	box-shadow: inset -0.375rem 0 0 -0.1875rem var(--sapList_SelectionBorderColor);
+}
+
+:host([navigated]) tr[data-ui5-dir="rtl"] {
+	box-sizing: border-box;
+	box-shadow: inset  0.1875rem 0 0 0 var(--sapList_SelectionBorderColor);
```

Comment:
There is an extra space between inset and 0.1875rem

Reply:
I removed the extra space.